### DISCUSSION TITLE
Rabbitmq package not available on packagecloud.io

### DISCUSSION THREAD

**Discussion by mvarchdev** [rabbitmq/rabbitmq-server#8387] [Other] [answered]
### Describe the bug

Cannot download Rabbitmq-server from packagecloud, getting "Payment required" error. I need it urgently for deploying Openstack-Ansible.

### Reproduction steps

1. Go to for example `https://packagecloud.io/rabbitmq/rabbitmq-server/ubuntu/dists/jammy/InRelease` and you will see the error

### Expected behavior

It should get package normally

### Additional context

Using it for deploying openstack-ansible

**Comment by michaelklishin** [comment, marked as answer]
https://github.com/rabbitmq/rabbitmq-server/discussions/8386
